_pages/governance.md

```diff
@@ -60,6 +60,11 @@ Published at the OpenAMP [code of conduct][code of conduct] page.
 
 # Development Process
 
+The development process in this section applies to the libmetal and open-amp libraries and any RTOS code in openamp-system-reference.
+The pull request model applies to all OpenAMP repositories, except from the kernel-staging repository
```

Comment:
If you look at the preview in https://production-openampproject-org-99.ghactions.linaro.org/governance/ you will see that this all becomes 1 paragraph.  So, need a period at the end of line 64.  Optional if you want to insert blank lines to get a new paragraph.

Reply:
Once the above is resolved, I'm OK with merging these changes.